src/sync/installation.ts

```diff
@@ -209,14 +209,221 @@ export const isNotFoundError = (
 	return isNotFoundError;
 };
 
+// TODO: type queues
+async function doProcessInstallation(app, queues, job, installationId: number, jiraHost: string, logger: LoggerWithTarget): Promise<void> {
```

Comment:
I don't think we need to pass `queues` as a function parameter. they are defined at the "../worker/queues" module as a global constant